package.json

```diff
@@ -26,17 +26,13 @@
     "node": "10"
   },
   "scripts": {
-    "db:add-data": "node scripts/addData",
-    "db:drop": "node scripts/dropDb",
-    "db:init": "run-s db:drop db:setup db:add-data",
-    "db:setup": "node scripts/setupDb",
     "format": "prettier --write \"**/*\"",
     "start": "run-p start:*",
     "start:api": "nodemon -q --watch src/api src/api/index.js",
     "start:app": "parcel src/app/index.html --open",
     "build:app": "parcel build src/app/index.html",
     "test": "run-p test:*",
-    "test:format": "prettier --check \"**/*\"",
+    "test:format": "prettier --check \"src/**/*\"",
```

Comment:
The was deliberately unspecific as it lets us format dot files like config

Reply:
will remove

Reply:
cool